app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -379,6 +383,10 @@ angular.module('dashboard')
         };
 
         self.changeMeetingStatus = function () {
+            if (!angular.isObject(mtgItem)) {
+                $log.error("meetingStatusCtrl.changeMeetingStatus: mtgItem missing");
+                return;
+            }
             $log.debug("meetingStatusCtrl.changeMeetingStatus");
             var items = [];
             angular.forEach(CONST.MEETINGSTATUSACTIONS, function (status) {
```

Comment:
"Old" code but minor style issue, should a `var` be used instead of the `status` function argument...just to clarify that the original array is not modified by the `status.disabled =` assignment?